### DISCUSSION TITLE
Query Loops filter by Meta Value/Custom Field

### DISCUSSION THREAD

**Discussion by rimager** [WordPress/gutenberg#42478] [Ideas]
I would love to be able to filter the Query Loop block by a custom field (full disclosure; I'm using ACF). So aside from filtering by taxonomy and category it would be amazing if I could add the filter by custom meta value, also.

**Comment by t-hamano** [comment]
Hi.
With `query_loop_block_query_vars` hook introduced in WordPress 6.1, you may be able to do what you want to do.
[Extending the Query Loop block | Block Editor Handbook](https://developer.wordpress.org/block-editor/how-to-guides/block-tutorial/extending-the-query-loop-block/#making-your-custom-query-work-on-the-front-end-side)
